Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -0,0 +1,1946 @@
+        return f'Network_IPV_{number}', {
+            f'networkIPV{number}List': address
+        }
+
+
+def check_args_create_rule(rule_type: str, address: List, from_address: str, to_address: str, number: int):
+    """ Validate the arguments of the function
+        Args:
+            rule_type: str - The type of the rule.
+            address: List - A list of addresses, if relevant.
+            from_address: str - The from address, if relevant.
+            to_address: str - The to address, if relevant.
+            number: int - The number of the addresses IP V.
+    """
+    if not address and not from_address and not to_address:
+        raise Exception('Please enter a matching address.')
+    if ('4' in rule_type and number == 6) or ('6' in rule_type and number == 4):
+        raise Exception('The version of the IP in "rule_object_type" should match the addresses version.')
+    if ('HOST' in rule_type or 'NETWORK' in rule_type) and (not address or from_address or to_address):
+        raise Exception(f'If the "rule_object_type" is “Endpoint IP V.{number}” or “Network IP V.{number}” than only'
+                        f' the argument “address_ip_v.{number}” must contain a value. The other address arguments '
+                        f'should be empty.')
+    if 'ADDRESS_RANGE' in rule_type and (not to_address or not from_address or address):
+        raise Exception(f'If the "rule_object_type" is “Range IP V.{number}” than only the arguments '
+                        f'“from_address_ip_v.{number}” and “to_address_ip_v.{number}” must contain a value, the other'
+                        f' address arguments should be empty.')
+
+
+def h_r_get_domains(children: List[Dict], human_readable: List):
+    """ Creates the human readable for the command get_domains.
+        Args:
+            children: List[Dict] - A list of the children.
+            human_readable: List - The human readable object.
+        Returns:
+            The human readable contains the relevant values.
+    """
+    for child in children:
+        child['ID'] = child.get('id')
+        del child['id']
+        child['Name'] = child.get('name')
+        del child['name']
+        d = {
+            'ID': child.get('ID'),
+            'Name': child.get('Name')
+        }
+        human_readable.append(d)
+        if child.get('childdomains', []):
+            h_r_get_domains(child.get('childdomains', []), human_readable)
+
+
+def update_source_destination_object(obj: List[Dict], rule_object_id: Optional[int], rule_object_type: Optional[str]) -> \
+        List[Dict]:
+    """ Updates the source and destination objects in the command update_firewall_policy.
+        Args:
+            obj: List[Dict] - The relevant object.
+            rule_object_id: Optional[int] - The id of the rule.
+            rule_object_type: Optional[str] - The type of the rule
+        Returns:
+            The updated object.
+    """
+    if rule_object_id:
+        if rule_object_id == -1:
```

Comment:
shouldn't we check here also the overwrite is not giving? or is false?

Reply:
I check if is_overwrite in the command, before calling the help function. This function (update_source_destination_object) is for is_overwrite=false, and overwrite_source_destination_object function is for is_overwrite=true.